lib/KSelect/KeenUiSelect.vue

```diff
@@ -552,6 +553,12 @@
           break;
         }
       }
+
+      // look for KSelects nested within modals
+      const allSelects = document.querySelectorAll('div.modal div.ui-select');
```

Comment:
I'd recommend adding a comment to `modal` element saying that it's being used by means of `div.modal` selector in the `KeenUiSelect` so that in the future, we prevent from this logic being silently broken in case someone removes or renames this class in `KModal` (I think that quite commonly people won't get an idea to check if a class is used in another component before refactoring it)

Reply:
added explanatory comments to both `KModal` & `KeenUiSelect` above where `class` is assigned within their templates - i'd love to hear what you think!